smallLibraries/Google.Cloud.Templates/NuGetPackage.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <PackageId>Google.Cloud.Templates</PackageId>
+    <PackageVersionPrefix>1.0.0</PackageVersionPrefix>
```

Comment:
I've been finding it simpler just to have a single version element - the whole prefix/suffix part has been only an annoyance (that I need to tidy up in some places)